### PR TITLE
Change the username from `MadhavBahlMD` to `MadhavBahl` #369

### LINKED ISSUE #369: Change the username from `MadhavBahlMD` to `MadhavBahl`
I have changed my username from `MadhavBahlMD` to `MadhavBahl`

Creating an issue to update the username at all places, 

assigning this issue to myself 

But I won't be taking this up anytime soon, So if anyone (beginner in open source) wants to take this documentation issue as a first time contribution, you can utilize this issue 😃

### PR DESCRIPTION
**Pull request by mihirg008**
# Please make sure you have done the following:

- [x] Updated README.md f
- [x] Added yourself as a contributor using `all-contributors-cli`
-this commit fixes #369

### CODE REVIEW

**Comment by MadhavBahl** [comment]
Thanks a lot @mihirg008 

I will review and merge soon

**Comment by mihirg008** [comment]
This is my first time contributing. Can you please add "hacktoberfest-accepted" label to this PR.

**Review by aaditkamat — APPROVED** [review]
Thank you for the contribution!

File: .all-contributorsrc [hunk 1]
```diff
@@ -328,6 +328,15 @@
...
       ]
+    },
+    {
+      "login": "mihirg008",
+      "name": "Mihir Gupta",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/32246337?s=460&u=26abf142607d487660a12fcc8b508407b0896c8c&v=4",
+      "profile": "https://github.com/mihirg008",
+      "contributions": [
+        "doc"
+      ]
```
Comment (aaditkamat) [inline .all-contributorsrc:339]: You can run the `npm run contributors:generate` command mentioned in the [Contributing document](https://github.com/CodeToExpress/dailycodebase/blob/master/CONTRIBUTING.md) to add your name to the [Contributors document](https://github.com/CodeToExpress/dailycodebase/blob/master/CONTRIBUTORS.md)

**Pull request merged**